jekyll/_ccie/quick-start.md

```diff
@@ -1,71 +1,81 @@
-experience extremely similar to using circleci.com, so much of the information in the
-[docs](https://circleci.com/docs/) and [support forum](https://discuss.circleci.com)
-for circleci.com are applicable. There are a few potentially
-confusing differences between CircleCI Enterprise and circleci.com though, so this guide
-provides a starting point specifically for users of Enterprise.
-
-Let's quickly cover a few of the first things you might want to try on CircleCI Enterprise. If you
-are curious about the differences between your CircleCI Enterprise installation and public circleci.com,
-you can skip to the [differences from circleci.com]({{site.baseurl}}/enterprise/differences/).
-
-<ul><li><b>Add your first project:</b> Setting projects up to build on CircleCI Enterprise is really easy.
-Just:
-<ol>
-    <li>Sign up for an account on your CircleCI Enterprise instance (make sure you have a GitHub Enterprise account first)</li>
-    <li>Select the GitHub user or organization of the project you want to build</li>
-    <li>Find your project in the list of projects and click "Build project" (or "Watch project" if it is already being built on CircleCI)</li>
-</ol>
-<p>That's it! If the inferred steps don't do quite what you want, try adding a `circle.yml` file as described below. You
-can add more projects at any time by clicking the "Add Projects" link on the left sidebar from the CircleCI Enterprise app.
-You can also check out a video walkthrough here:</p>
+This guide describes how to set up a project and provides a primer on configuration troubleshooting in CircleCI Enterprise for developers, as follows:
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+To add a project, you must have a GitHub or GitHub Enterprise account.
+
+## Adding a Project
+
+Complete the following steps to set up a project on CircleCI Enterprise.
+
+1. Click the Open link on the https://localhost:8800/dashboard page, or navigate to the IP address for the CircleCI application as installed by your administrator. The Welcome page appears with a Get Started button. ![CircleCI Enterprise Welcome]({{ site.baseurl }}/assets/img/docs/ccie_welcome.png)
+2. Click the Get Started button on the Welcome to CircleCI page. If you are not already logged into GitHub, the Sign in to GitHub to continue to Local CCIE Enterprise page appears. ![CircleCI Enterprise Login]({{ site.baseurl }}/assets/img/docs/ccie_login.png)
+3. Enter your GitHub login credentials and click the Sign in button followed by your Two-factor authentication if appropriate. The CircleCI application appears with a Getting Started section. ![CircleCI Enterprise Get Started]({{ site.baseurl }}/assets/img/docs/ccie_follow_build.png)
+4. Click the Follow and Build button to start building your projects and get email notifications of success or failure. The Builds page appears with your first project in the Running state. ![CircleCI Enterprise Running Build]({{ site.baseurl }}/assets/img/docs/ccie_builds_running.png)
```

Comment:
Consider adding an instruction to select the projects they want to follow rather than assuming clicking straight through is the best option.